src/main/java/com/google/api/codegen/transformer/py/PythonApiMethodParamTransformer.java

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2017 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.py;
+
+import com.google.api.codegen.transformer.ApiMethodParamTransformer;
+import com.google.api.codegen.transformer.MethodTransformerContext;
+import com.google.api.codegen.viewmodel.DynamicLangDefaultableParamView;
+import com.google.api.codegen.viewmodel.ParamDocView;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+public class PythonApiMethodParamTransformer implements ApiMethodParamTransformer {
+  @Override
+  public List<DynamicLangDefaultableParamView> generateMethodParams(
+      MethodTransformerContext context) {
+    // TODO(eoogbe): implement this method when migrating to MVVM
+    return ImmutableList.<DynamicLangDefaultableParamView>of();
```

Comment:
The methods get called in `DynamicLangApiMethodTransformer`, just the values don't get used. Need to return dummy values, not throw an exception.